packages/puppeteer-core/src/common/bidi/Realm.ts

```diff
@@ -155,9 +168,7 @@ export class Realm extends EventEmitter {
       responsePromise = this.connection.send('script.callFunction', {
         functionDeclaration,
         arguments: await Promise.all(
-          args.map(arg => {
-            return BidiSerializer.serialize(arg, this as any);
-          })
+          args.map(BidiSerializer.serialize.bind(BidiSerializer, sandbox))
```

Comment:
The `BidiSerializer` is a static class so would prefer to keep the function as you are never using `this`